Sources/ArgumentParser/Parsable Properties/ArgumentVisibility.swift

```diff
@@ -10,7 +10,22 @@
 //===----------------------------------------------------------------------===//
 
 /// Visibility level of an argument's help.
-public enum ArgumentVisibility {
+public struct ArgumentVisibility {
+    let rawValue: _ArgumentVisibility
+
+    /// Show help for this argument whenever appropriate.
+    public static let `default` = Self(rawValue: .default)
+
+    /// Only show help for this argument in the extended help screen.
+    public static let hidden = Self(rawValue: .hidden)
+
+    /// Never show help for this argument.
+    public static let `private` = Self(rawValue: .private)
+}
+
+/// Internal implementation of `ArgumentVisibility` to allow for easier API
+/// evolution
+internal enum _ArgumentVisibility {
```

Comment:
Let's move this to be nested inside `ArgumentVisibility` and rename to match the other types that are set up like this, e.g. https://github.com/apple/swift-argument-parser/blob/main/Sources/ArgumentParser/Parsable%20Properties/Flag.swift#L126-L132

Reply:
Should be fixed.